web/react/components/create_comment.jsx

```diff
@@ -184,6 +184,7 @@ module.exports = React.createClass({
                 </div>
             );
         }
+        var allowTextFormatting = config.AllowTextFormatting;
```

Comment:
Using the client config is fine for now but in the future we'll want this coming from the server (can be bundled in with the ticket to put all config on the server)